demo-deploy/README.md

```diff
@@ -0,0 +1,53 @@
+# Infrastructure Deployment Makefile
+
+This repository includes a `Makefile` to streamline the deployment, management, and teardown of AWS infrastructure using Terragrunt. The script ensures all prerequisites are met and simplifies executing commands for planning, applying, and destroying infrastructure across all modules in a specific environment.
```

Comment:
The sentence "The script ensures all prerequisites are met and simplifies executing commands for planning, applying, and destroying infrastructure across all modules in a specific environment." looks redundent and repeats the first sentance.

I'm mentioning this because most devs will read only the first paragraph of the docs so it better be good.